### ISSUE TITLE
[DOCS] default master_type of str has no description

### ISSUE THREAD

**Issue by bugfood** [saltstack/salt#61963]
**Description**
Default `master_type: str` not documented. The fact that `str` is default _is_ documented, but there is no explanation of what the `str` behavior is.

**Suggested Fix**
Please document what `str` does. This appears to mean "connect to all masters". I don't know what `str` is an abbreviation for.

**Type of documentation**
Salt documentation

**Location or format of documentation**
https://docs.saltproject.io/en/latest/ref/configuration/minion.html#master-type

**Comment by OrangeDog** [comment]
`str` means the Python type `str`

> The type of the [master](https://docs.saltproject.io/en/latest/ref/configuration/minion.html#std-conf_minion-master) variable

These two parameters are documented poorly though. They should be together, with an example of both set for each type.

**Comment by bugfood** [comment]
Yes, I don't think "The type of the [master] variable" corresponds to current behavior at all; the default is just set to `str` (the literal string) regardless of the variable type of `master`, and the other documented values of `master_type` are not python variable types. I'm rather confused here. :)

**Comment by dwoz** [comment]
Already documented on all supported branches (3006.x, 3007.x, 3008.x, master) by commit eb0bd49d4c2 ("docs(minion):: document master_type: str"). Current docs read: "If this option is `str` (default), multiple hot masters are configured. Minions can connect to multiple masters simultaneously (all master are `hot`)." Closing.
